ts/src/header-validator/validate-json.ts

```diff
@@ -1279,28 +1279,54 @@ function aggregatableTriggerData(
   )
 }
 
+export type AggregatableValuesValue =
```

Comment:
To match [the spec](https://pr-preview.s3.amazonaws.com/agarant/attribution-reporting-api/pull/1290.html#aggregatable-key-value-filtering-id), I think this should remove the post-parsing distinction between ones with a `filtering_id` and ones without.